.gitignore

```diff
@@ -12,6 +12,7 @@
 .\#*\#
 .vs
 .clang-format
+.*
```

Comment:
won't that make git ignore all `.git*` files as well as the `.ci` and `.travis.yml` files?

Reply:
Files that are already tracked are not considered by .gitignore.

Reply:
ok, then the two lines are useless right?

Reply:
Yes.